docs/Configuration.md

```diff
@@ -59,7 +59,7 @@ The JsonPathContentTemplate allows matching on and extracting values from an Eve
 |**TypeMatchExpression**|The JSON Path expression that is evaluated against the EventData payload. If a matching JToken is found the template is considered a match. All subsequent expressions are evaluated against the extracted JToken matched here.|`$..[?(@heartRate)]`
 |**TimestampExpression**|The JSON Path expression to extract the timestamp value for the measurement's OccurenceTimeUtc.|`$.endDate`
 |**DeviceIdExpression**|The JSON Path expression to extract the device identifier.|`$.deviceId`
-|**PatientIdExpression**|*Optional*: The JSON Path expression to extract the patient identifier.|`$.patientId`
+|**PatientIdExpression**|*Required* when IdentityResolution is in **Create** mode and *Optional* when IdentityResolution is in **Lookup** mode. The JSON Path expression to extract the patient identifier.|`$.patientId`

```

Comment:
Nit: would that be possible to put the link to where we explain the difference between "Create" and "Lookup", it should be convenient for customer to jump to right place to learn or review. It took me a while to understand the difference and prerequisites for "Lookup" in navigating the pages so I thought it would be also helpful to the customer as well.